modules/_labs/color-picker/react/stories/stories_ColorPicker.tsx

```diff
@@ -5,7 +5,7 @@ import {action} from '@storybook/addon-actions';
 import withReadme from 'storybook-readme/with-readme';
 import {ColorInput} from '@workday/canvas-kit-react-color-picker';
 import {colors} from '@workday/canvas-kit-react-core';
-import {Popper} from '@workday/canvas-kit-react-common';
+import {Popper} from '@workday/canvas-kit-react-popup';
```

Comment:
`Popper` is moving from common to popup. This is documented in the migration guide. Docs have been updated